CMake/HPHPSetup.cmake

```diff
@@ -165,6 +165,14 @@ add_definitions(-DDEFAULT_CONFIG_DIR="${DEFAULT_CONFIG_DIR}")
 
 add_definitions(-DHAVE_INTTYPES_H)
 
+# Load system tzdata if present
```

Comment:
No idea if this is the best place to do this. I copied this code from `third-party/forks/timelib/CMakeLists.txt`. Please feel free to suggest an alternate location for this. It had to move somewhere that impacts hhvm's own compilation though because it is now being used in `timezone.cpp` in hhvm.

Also we discussed that we potentially should just bail and not compile here if this isn't present but I'll leave that up to you to decide.